services/nutanix-ai/metadata.yaml

```diff
@@ -0,0 +1,21 @@
+displayName: Nutanix AI
+description: NAI core components.
+allowMultipleInstances: false
+category:
+  - Data Services
+certifications:
+  - certified
```

Comment:
pls see more names here https://docs.google.com/document/d/1TFvMb8xlEckWDzt4DrVhqENEiwAIefkxTt2loY2LL3Q/edit#bookmark=id.kdal4251v7wo
we should add supported too

Reply:
yes